src/test/java/seedu/address/commons/util/ConfigUtilTest.java

```diff
@@ -76,8 +76,8 @@ private Config getTypicalConfig() {
         config.setAppTitle("Typical App Title");
         config.setLogLevel(Level.INFO);
         config.setUserPrefsFilePath("C:\\preferences.json");
-        config.setAddressBookFilePath("addressbook.xml");
-        config.setAddressBookName("TypicalAddressBookName");
+        config.setTaskManagerFilePath("addressbook.xml");
```

Comment:
It's reading and matching values from json files and not used in anywhere else. So for now, I'll just leave them hardcoded. I fixed that the rest as you suggested! Thanks for catching all these.